### PR TITLE
Update of implementation registers

### PR DESCRIPTION
**Pull request by onthebreeze**
Added BC Gov EMDT, BC Gov OrgBook, UNCEFACT reference implementation, Spherity, Trust provenance

### CODE REVIEW

**Review by zachzeus — APPROVED** [review]

File: website/docs/implementations/Registers.md [hunk 1]
```diff
@@ -17,23 +17,26 @@ Register types are
...
+* Participate in pre-release UNTP Pilots? : Yes
 
-** Organisation Information **
-
-|Logo|Name|Implementation Statement|Registration Date|Country|
-|--|--|--|--|--|
-|**LOGO**|Sample Register (URL hyperlink)|As a global register of trade products we will implement UNTP so that anyone that encounters a registered product (eg a barcode) can discover rich and verifiable sustainability information about that product|01-Jan-2025|AU|
+|Logo|Implementation Statement|
+|--|--|
+|![BC Gov](../../implementations/orgbook.gov.bc.ca/logo.png)|The BC government sees UNTP implementation as an opportunity to enable BC producers of raw materials to differentiate their products in emerging sustainability-focused markets, contributing to a sustainable, clean, secure, and fair global economy. |
```
Comment (bblazicevic) [inline website/docs/implementations/Registers.md:35]:
```suggestion
|![BC Gov](../../implementations/orgbook.gov.bc.ca/logo.png)|The Government of British Columbia (BC) sees UNTP implementation as an opportunity to enable BC producers of raw materials to differentiate their products in emerging sustainability-focused markets, contributing to a sustainable, clean, secure, and fair global economy. |
```

File: website/docs/implementations/Registers.md [hunk 2]
```diff
@@ -17,23 +17,26 @@ Register types are
...
 
 ** Registered Identifier Schemes**
 
-|Regulation|Description|UNTP Scope|UNTP Version(s)|Implementation Date|Test Report|
-|--|--|--|--|--|--|
-|Product ID|Unique identifier for a product class|IDR, DIA|v0.5.0|01-Feb-2025|TBA |
+|Scheme|Description|UNTP Scope|UNTP Version(s)|Implementation Date|Test Report|
+|--|--|--|--|--|--|--|
+|[BC Org Book](https://orgbook.gov.bc.ca/)|Public register of organisations registered in BC|DCC|v0.5, v1.0|TBA|TBA |
```
Comment (bblazicevic) [inline website/docs/implementations/Registers.md:41]:
```suggestion
|[BC Org Book](https://orgbook.gov.bc.ca/)|Public verifiable register of organisations registered in Brish Columbia|DCC|v0.5, v1.0|TBA|TBA |
```

File: website/docs/implementations/Regulators.md [hunk 3]
```diff
@@ -13,23 +13,27 @@ Summary list of regulators (ie international bodies, national government agencie
 
 |Organisation Name |Regulatory Function|UNTP Scope|Status|
 |--|--|--|--|
-|[Sample Agency](#sample-agency)|Environmental protection|DCC|planned| 
+|[Government of British Columbia](#government-of-british-columbia)|Energy and Mines|DCC|planned|
```
Comment (bblazicevic) [inline website/docs/implementations/Regulators.md:16]:
```suggestion
|[Government of British Columbia](#government-of-british-columbia)|Energy and Mines Digital Trust|DCC|planned| 
```

**Review by bblazicevic — APPROVED** [review]
great work!

**Pull request merged**